python_scripts/02_numerical_pipeline_cross_validation.py

```diff
@@ -0,0 +1,151 @@
+# %% [markdown]
```

Comment:
I believe this notebook will need the jupyter metadata header for jupyter sync to work.

Reply:
Done, thanks for the catch!